pkg/helm/helm.go

```diff
@@ -257,6 +257,16 @@ func (c *Client) Install(ctx context.Context, chart string, opts InstallOptions)
 	return c.Get(ctx, opts.Namespace, result["name"].(string))
 }
 
+func (c *Client) Uninstall(ctx context.Context, release Release) error {
+	args := []string{"uninstall", release.Name, fmt.Sprintf("--namespace=%s", release.Namespace), "--wait", "--cascade=foreground"}
+
+	_, _, err := c.runHelm(ctx, args...)
+	if err != nil {
+		return err
+	}
+	return err
```

Comment:
nit: `return nil` to be explicit. I'd expect staticlint to yell at you for this.

Reply:
Ok, will do that.